src/include/stir/KOSMAPOSL/KOSMAPOSLReconstruction.h

```diff
@@ -0,0 +1,291 @@
+//
+//
+/*
+    Copyright (C) 2000 PARAPET partners
```

Comment:
I think it would be fair to remove copyright statements upto 2012 (and the PARAPET one). There is hardly (no?) code left from OSMAPOSL.